frontend/src/common/authentication/OnRouteBCContext.tsx

```diff
@@ -16,7 +16,7 @@ export type IDIRUserDetailContext = {
   lastName: string;
   userName: string;
   email: string;
-  userAuthGroup: IDIRUserAuthGroupType;
+  userRole: IDIRUserAuthGroupType;
```

Comment:
Should we rename "IDIRUserAuthGroupType" and "BCeIDUserAuthGroupType" to "IDIRUserRole" and "BCeIDUserRole"?